zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/deployment/DbFormState.java

```diff
@@ -170,6 +206,14 @@ public void deleteFormInFormKeyByFormIdAndDeploymentKeyColumnFamily(final FormRe
     formKeyByFormIdAndDeploymentKeyColumnFamily.deleteIfExists(tenantAwareFormIdAndDeploymentKey);
   }
 
+  @Override
+  public void deleteFormInFormKeyByFormIdAndVersionTagColumnFamily(final FormRecord record) {
+    tenantIdKey.wrapString(record.getTenantId());
+    dbFormId.wrapString(record.getFormId());
+    dbVersionTag.wrapString(record.getVersionTag());
+    formKeyByFormIdAndVersionTagColumnFamily.deleteIfExists(tenantAwareFormIdAndVersionTagKey);
+  }
+
```

Comment:
🔧 It would be nicer to put this change in a separate commit, for better understanding of the changes in the future.

Reply:
I've extracted the deletion related changes into a separate [commit](https://github.com/camunda/camunda/pull/21494/commits/4a1c4f6bd7bb26a8c5e7b8d09dd5acca14c7ef8c).